examples/linux/policyfile_cookbook/metadata.rb

```diff
@@ -0,0 +1,12 @@
+name 'policyfile_cookbook'
+maintainer 'Jerry Aldrich'
```

Comment:
It's probably a good idea to remove metadata from this file.

Reply:
Sure, what should I put in its place? It is a full example cookbook to be used as reference, though nothing really changes in this file in regards to Habitat stuff.

Reply:
You'll need a version for a valid metadata.rb right?

I don't think you need the maintainer line for the cookbook to function.

In the effortless model all the metadata lives in the plan.sh / plan.ps1, so ripping out all this boilerplate from the cookbook level will prevent double-writing metadata all over the place.

Reply:
Removing the metadata and LICENSE causes `foodcritic` to fail. 🤔 

Not sure if we should remove that, but I totally see your point about duplicated metadata.

Reply:
We shouldn't care about `foodcritic`, only `chefstyle`. IMO

Reply:
My concerns are around:
  - Pipelines for cookbooks using foodcritic (probably solvable with excludes)
  - Uploading to Supermarket so other cookbooks can consume this cookbook via Policy/Hab
    - I think those foodcritic rules exist for Supermarket only.

Reply:
I guess we can include it for now. Let's go.